### ISSUE TITLE
Suggestion: Display date created

### ISSUE THREAD

**Issue by exmqyu526j** [keybase/keybase-issues#2582]
It could be valuable to publicly display the date an account was created, as well as the date a key was added.  This could help rule out fake accounts.

**Comment by zQueal** [comment]
This seems like a pretty common sense security protocol which has gone largely overlooked. I'd totally throw my support behind this. Maybe as a simple tool-tip (to keep it out of the way and keep the interface clean) while hovering over the orange username in the `keybase.io/` introduction text?

![ybmtil](https://cloud.githubusercontent.com/assets/239193/18773357/6aa5cd24-811c-11e6-8db2-794ce46eb18b.png)

**Comment by exmqyu526j** [comment]
Yes, I think that's a great idea.  You could make a function for the tool-tip, and use it on anything that would benefit from it.  I imagine the information is already stored in the database, so it shouldn't be difficult at all.

**Comment by malgorithms** [comment]
we'll think about this sometime soon on the design side. the info is there, of course...all signed statements on keybase have timestamps on them and fit into a data structure with other people's signed statements. You can confirm he created he's had the same signature chain since March, 2014 (when he reset his account). 

https://keybase.io/zqueal/sigchain#9d4be5dc3cba9adece0f7210a8516425ee822fa4e9b26b3ff814b948bac6bcc10f

you can also see when all the individual proofs were posted.
